src/Settings.php

```diff
@@ -183,6 +183,11 @@ public function save(): self
         return $this;
     }
 
+    public function update($properties): self
```

Comment:
you should probably type `$properties` as `array`

Reply:
Indeed